src/legacy/state/orders/updaters/UnfillableOrdersUpdater.ts

```diff
@@ -144,7 +146,7 @@ export function UnfillableOrdersUpdater(): null {
         const currencyAmount = CurrencyAmount.fromRawAmount(order.inputToken, order.sellAmount)
         const enoughBalance = hasEnoughBalanceAndAllowance({ account, amount: currencyAmount, balances })
 
-        _getOrderPrice(chainId, order, enoughBalance, strategy)
+        _getOrderPrice(chainId, order, enoughBalance && verifiedQuotesEnabled, strategy)
```

Comment:
Can you use the flag to short circuit the balance check and avoid some extra calls when it's off?